CONTRIBUTING.md

```diff
@@ -7,6 +7,8 @@ This guideline intends to make contribtuions to Caliper easier by:
 
 If you have further suggestions about improving the guideline, then you can follow the presented workflow to make your contribution.
 
+Jump to [Contributing documentation](./docs/Contributing.md) to get started on your journey.
```

Comment:
This is the wrong link, when I click on it I get a 404 not found. It's docs/source/contributing.md

however I am wondering whether this should link to the md file or whether it should go to the served docs (ie https://hyperledger.github.io/caliper/). For now it works. We should probably change it once we have the new docs published as the served docs presents a nicer view than just the standard md viewer.

Reply:
Yeah, after deployment I will change the link to the served docs